docs/extending/creating_custom_dropdowns.md

```diff
@@ -133,10 +122,12 @@ You can configure the following attributes:
 
 ![Drop-down expanded state](img/dropdown_expanded_state.png)
 
+# Extend drop-down templates
+
 ## Initialize
 
 All drop-downs are searched and initialized automatically in `admin.dropdown.js`.
-To extend or modify the search, you need to add a `custom_init` parameter to the Twig element. Otherwise it will be initialized two times.
+To extend or modify the search, you need to add a `custom_init` attribute to the drop-down Twig parameteres. Otherwise it will be initialized two times.
```

Comment:
```suggestion
To extend or modify the search, you need to add a `custom_init` attribute to the drop-down Twig parameters. Otherwise it will be initialized two times.
```